### DISCUSSION TITLE
Tailwind starter repo command does not work

### DISCUSSION THREAD

**Discussion by Puneet56** [nandorojo/solito#145] [Q&A] [answered]
The command
`npx create-solito-app@latest my-solito-app -t with-tailwind`
creates solito template instead of tailwind one.
![image](https://user-images.githubusercontent.com/26213419/189476338-1d494854-b56a-47d7-ab18-55f678c89716.png)
![image](https://user-images.githubusercontent.com/26213419/189476340-aecc1785-a771-4ac9-b0ea-6a317902fb9e.png)

**Comment by nandorojo** [comment, marked as answer]
well that’s weird. i’ll look into this. thanks!

**Reply by nandorojo** [reply]
should be fixed now with #147
